.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,6 +18,19 @@ $ spack <command2> <spec>
 ...
 ```
 
+### Error Message
+
+What you expected before running the command or performing the steps 
```

Comment:
I think 

> What you expected before running the command or performing the steps described above and what happened instead. To provide more information you might re-run the commands with the additional `-sd` flags:

Could be rephrased:

> If Spack reported an error, provide the error message. If it did not report an error but the output appears incorrect, provide the incorrect output. If there was no error message and no output but the result is incorrect, describe how it does not match what you expect.

(given that the summary asks the user to describe what they expect)